api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityService.java

```diff
@@ -18,10 +18,11 @@
     private final ClaimRepository claimRepository;
 
     /**
-     * Determines the eligibility for a given status. If the claimant's nino is not found in the database,
+     * Determines the eligibility for a given claimant. If the claimant's NINO is not found in the database,
      * the external eligibility service is called.
+     *
      * @param claimant the claimant to check the eligibility for
-     * @return an eligibility response for the claimant
+     * @return the eligibility response for the claimant
      */
     public EligibilityResponse determineEligibility(Claimant claimant) {
```

Comment:
Should we rename this method to state it's for new claims?

Reply:
Yep, done.